setup.go

```diff
@@ -44,15 +44,15 @@ func setup(c *caddy.Controller) error {
 			return fmt.Errorf("%s is not a valid rcode", rcode)
 		}
 
-		u, err := proxy.NewStaticUpstream(&c.Dispenser)
+		u, err := forward.ParseForwardStanza(&c.Dispenser)
 		if err != nil {
```

Comment:
I guess this **ParseForwardStanza** does not exist yet in plugin/forward.
Should you wait another PR on coredns ?

Reply:
I see the PR https://github.com/coredns/coredns/pull/2034